vunit/sim_if/xcelium.py

```diff
@@ -0,0 +1,427 @@
+        cds.write(self._cdslib)
+
+    def _get_mapped_libraries(self):
+        """
+        Get mapped libraries from cds.lib file
+        """
+        cds = CDSFile.parse(self._cdslib)
+        return cds
+
+    def simulate(  # pylint: disable=too-many-locals, too-many-branches
+        self, output_path, test_suite_name, config, elaborate_only=False
+    ):
+        """
+        Elaborates and Simulates with entity as top level using generics
+        """
+
+        script_path = str(Path(output_path) / self.name)
+        launch_gui = self._gui is not False and not elaborate_only
+
+        if elaborate_only:
+            steps = ["elaborate"]
+        else:
+            steps = ["elaborate", "simulate"]
+
+        for step in steps:
+            cmd = str(Path(self._prefix) / "xrun")
+            args = []
+            if step == "elaborate":
+                args += ["-elaborate"]
+            args += ["-nocopyright"]
+            args += ["-licqueue"]
+            # args += ['-dumpstack']
+            # args += ['-gdbsh']
+            # args += ['-rebuild']
+            # args += ['-gdb']
+            # args += ['-gdbelab']
+            args += ["-errormax 10"]
+            args += ["-nowarn WRMNZD"]
+            args += ["-nowarn DLCPTH"]  # "cds.lib Invalid path"
+            args += ["-nowarn DLCVAR"]  # "cds.lib Invalid environment variable ''."
+            args += [
+                "-xmerror EVBBOL"
+            ]  # promote to error: "bad boolean literal in generic association"
+            args += [
+                "-xmerror EVBSTR"
+            ]  # promote to error: "bad string literal in generic association"
+            args += [
+                "-xmerror EVBNAT"
+            ]  # promote to error: "bad natural literal in generic association"
+            args += ["-work work"]
+            args += [
+                '-xmlibdirname "%s"' % (str(Path(self._output_path) / "libraries"))
+            ]  # @TODO: ugly
+            args += config.sim_options.get("xcelium.xrun_sim_flags", [])
+            args += ['-cdslib "%s"' % self._cdslib]
+            args += self._hdlvar_args()
+            args += ['-log "%s"' % str(Path(script_path) / ("xrun_%s.log" % step))]
+            if not self._log_level == "debug":
+                args += ["-quiet"]
+            else:
+                args += ["-messages"]
+                # args += ['-libverbose']
+            args += self._generic_args(config.entity_name, config.generics)
+            for library in self._libraries:
+                args += ['-reflib "%s"' % library.directory]
+            args += ['-input "@set intovf_severity_level ignore"']
+            if config.sim_options.get("disable_ieee_warnings", False):
+                args += [
+                    '-input "@set pack_assert_off { std_logic_arith numeric_std }"'
+                ]
+            args += [
+                '-input "@set assert_stop_level %s"' % config.vhdl_assert_stop_level
+            ]
```

Comment:
I took it from #684 but I'm not sure it's necessary. If yes, must be also applied to Incisive?

Reply:
```
intovf_severity_level = value
The value of this variable determines whether an integer overflow violation is reported as an error,
which stops the simulation, as a warning, or if the violation is ignored.
```
```
pack_assert_off = value
The value of this variable specifies the package name(s) from which you want to suppress the
display of VHDL assert messages.
```
```
assert_stop_level = value
Specifies the minimum severity level for which VHDL assertions cause the simulation to stop.
The value can be note , warning , error , failure , or never .
This variable is initially set to error .
```
Seems ok.

Reply:
Yes, these also apply to Incisive.